js/client-teambuilder.js

```diff
@@ -355,13 +355,19 @@
 				}
 			}
 
-			var newButtonText = "New Team";
-			if (filterFolder) newButtonText = "New Team in folder";
+			var newTeamButtonText = "New Team";
+			var newBoxButtonText = "New Box";
+			if (filterFolder) {
+				newTeamButtonText = "New Team in folder";
+				newBoxButtonText = "New Box in folder";
```

Comment:
One thing we could do is just display "New Box" and hope being next to the "New Team" button is clear enough.